src/Storage/Token.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * This file is part of Lcobucci\JWT, a simple library to handle JWT and JWS
+ *
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
+ */
+
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Storage;
```

Comment:
Why was `Token` moved down to `Storage`? There is no real storage stuff going on anywhere in this PR, as far as I can see...